protos/docs/datacatalog/index.rst

```diff
@@ -1,10 +1,10 @@
 Flyte Data Catalog Service
-=================================
+============================
 
-These protos provide the interface definition for the Data Catalog Service. Data Catalog is a service for
+Protos provide the interface definition for the Data Catalog Service. Data Catalog is a service for
 indexing parameterized, strongly-typed data artifacts across revisions. It is used in the Flyte ecosystem
-to catalog artifacts generated by task executions. Outputs generated by a task can be stored as artifact
-data and tagged by the user so that it can be retrieved later by that tag.
+to catalog artifacts generated by the task executions. Outputs generated by a task can be stored as artifact
```

Comment:
```suggestion
to catalog artifacts generated by the task executions. Outputs generated by a task can be stored as an artifact
```

Reply:
Should it be 'an artifact data' or 'artifact data' or 'an artifact datum'? im confused. what would you suggest? @AlekhyaSasi

Reply:
I don't think "an" is required. We can keep "artifact data" so that we need not worry about what goes before it.